test/complex.jl

```diff
@@ -925,7 +932,7 @@ end
     @test Float16(1)+Float16(1)im === Complex32(1, 1)
     @test Float16(1)-Float16(1)im === Float16(1)+Float16(-1)im === Complex32(1, -1)
     @test Float16(1)*im === Complex32(im)
-    @test Float16(1)/im === 1.0f0/im === Complex(0.0, -1.0)
+    @test Float16(1)/im === Complex32(0,-1) == -Float16(1)*im
```

Comment:
actually, it was wrong (`-Float16(1)*im === Complex32(-0.0,-1.0)`); I've removed it.